controller/scheduler/states.py

```diff
@@ -1,9 +1,14 @@
 import enum
 
 
-class TransitionNotAllowed(Exception):
+class TransitionError(Exception):
```

Comment:
I saw documentation in python https://docs.python.org/2/tutorial/errors.html#user-defined-exceptions and thought it was better